atom/browser/native_window_mac.h

```diff
@@ -213,6 +215,9 @@ class NativeWindowMac : public NativeWindow,
   NSRect original_frame_;
   NSUInteger simple_fullscreen_mask_;
 
+  NSColor* background_color_before_vibrancy_ = nil;
```

Comment:
I'm not positive but should this be a `base::scoped_nsobject<NSColor>`?

Reply:
You right 👍 😄